ui/core/components/individual_sim_ui/apl_values.ts

```diff
@@ -1129,38 +1128,62 @@ const valueKindFactories: { [f in ValidAPLValueKind]: ValueKindConfig<APLValueIm
 		label: 'Aura Active',
 		submenu: ['Aura'],
 		shortDescription: '<b>True</b> if the aura is currently active, otherwise <b>False</b>.',
-		newValue: APLValueAuraIsActive.create,
-		fields: [AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'), AplHelpers.actionIdFieldConfig('auraId', 'auras', 'sourceUnit')],
+		newValue: () => APLValueAuraIsActive.create({ includeReactionTime: true }),
+		fields: [
+			AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'),
+			AplHelpers.actionIdFieldConfig('auraId', 'auras', 'sourceUnit'),
+			AplHelpers.booleanFieldConfig('includeReactionTime', 'Include Reaction Time', {
+				labelTooltip: 'If checked, will use the configured reaction time.',
+			}),
+		],
 	}),
 	auraIsActiveWithReactionTime: inputBuilder({
-		label: 'Aura Active (with Reaction Time)',
+		label: '[DEPRECATED] Aura Active (With Reaction Time)',
 		submenu: ['Aura'],
-		shortDescription:
-			'<b>True</b> if the aura is currently active AND it has been active for at least as long as the player reaction time (configured in Settings), otherwise <b>False</b>.',
```

Comment:
Keep this as the `shortDescription` for the new `auraIsActive` implementation in order to match the description for `auraIsInactive`.